src/App/App.js

```diff
@@ -137,6 +116,27 @@ class App extends Component {
   return changedData;
  }
 
+ login = async (loginState) => {
+  const response = await API.postData(loginState);
+  const user = await response.json();
+  if (response.status === 201) {
+   this.setState({
+    pageView: "Home",
```

Comment:
We'll get into an alternative option for this kind of logic tomorrow with Router, make sure to think about how the next lesson can be used to refactor your code.